scripts/grafana/install

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+
+URL="https://bintray.com/fg2it/deb/download_file?file_path=main%2Fg%2Fgrafana_4.3.2_armhf.deb"
+
+if $(uname -m | grep -Eq ^armv6); then
+    URL="https://bintray.com/fg2it/deb-rpi-1b/download_file?file_path=main%2Fg%2Fgrafana_4.3.2_armhf.deb"
+fi
+
+sudo apt-get -y install libfontconfig1 fontconfig-config fonts-dejavu-core ttf-bitstream-vera fonts-liberation sqlite3
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Download and install IPFS
+mkdir "$BASE_DIR/tmp"
+
+wget ${URL} -O "$BASE_DIR/tmp/go-grafana.tar.gz"
+
+sudo dpkg -i "$BASE_DIR/tmp/go-grafana.tar.gz"
+
+rm -rf "$BASE_DIR/tmp"
+
+
+sudo /bin/systemctl daemon-reload
+sudo /bin/systemctl enable grafana-server
+sudo /bin/systemctl start grafana-server
+
+sleep 2 #Wait for it to start
```

Comment:
Yes be but i want to make sure the database is started and initialized which happens upon daemon start to shoe horn the data source in there..

i'm not sure if it forked before or after the creation of the database